apps/ledger-live-desktop/src/renderer/components/Box/Box.js

```diff
@@ -35,11 +35,9 @@ const Box: ThemedComponent<{
   relative?: boolean,
   sticky?: boolean,
   selectable?: boolean,
-  // DROP this when we fully migrated from V1
-  align?: "THIS PROPERTY IS NOT VALID – SWITCH TO alignItems",
-  justify?: "THIS PROPERTY IS NOT VALID – SWITCH TO justifyContent",
```

Comment:
unclear why this is related to this PR. maybe send a separate PR?

Reply:
Got this from the previous PR, will remove yes 👌